src/planner/add_hashagg.c

```diff
@@ -27,7 +27,7 @@
 #include "estimate.h"
 
 /* This optimization adds a HashAggregate plan to many group by queries.
- * In plain postgres, many time-series queries will not use a the hash aggregate
```

Comment:
The typo is still not fixed :-)

Reply:
It is for me are you looking at stale version?